lib/checkstl.cpp

```diff
@@ -534,6 +536,18 @@ void CheckStl::iterators()
     }
 }
 
+void CheckStl::mismatchingContainerIteratorError(const Token* tok, const Token* iterTok)
+{
+    const std::string container(tok ? tok->expressionString() : std::string("v1"));
+    const std::string iter(iterTok ? iterTok->expressionString() : std::string("v2"));
```

Comment:
nit; I suggest that the iterator name is changed from `v2` to `it`. there is nothing wrong here, just think that would be a bit more clear. I am not sure if anybody looks at this message at all.